demo/app/components/drawer/drawer.component.scss

```diff
@@ -13,3 +13,22 @@
     margin-right: 20px;
   }
 }
+
+.ts-drawer-header{
```

Comment:
IMO we should probably leave styling more default so that users can see what they get out of the box. Open to thoughts here though..